### PR TITLE
Real metric names

### PR DESCRIPTION
**Pull request by swellander**
## Description
Updates our metric names and OSO data to be in sync with the [current version of OSO's API](https://models.opensource.observer/#!/model/model.opensource_observer.rf4_impact_metrics_by_project#columns). Running a yarn seed --reset is needed for this branch to work locally

## Additional Information
- Updates our `seed.sample.json` with the new metrics
- Removes our references to the `mapping` object, that we no longer need because OSO includes an `application_id` field that is also the project's ID on Agora.
- Changes dummy data values to be more random than progressive, trying to mimic the time-bound metric values [that we will be showing](https://github.com/BuidlGuidl/onchain-impact-dashboard/issues/93)
- We'll need to decide which metrics to show in the graph on the front end. Right now, there's not enough space to show all metrics using their full names (see screenshot). Either that, or shorten the metric names
image

### CODE REVIEW

File: packages/nextjs/app/types/OSO.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export interface OnchainMetricsByProject {
   medium_activity_address_count_90_days: number;
   multi_project_address_count_90_days: number;
   new_address_count_90_days: number;
-  project_id: string;
+  application_id: string;
```
Comment (swellander) [inline packages/nextjs/app/types/OSO.ts]: This is the same as the project's Agora ID

File: packages/nextjs/pages/api/etl/impact_metric_weights.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
         "recurring_addresses": 5.163663192473145,
         "openrank_trusted_users_count": 4.054655175020334,
         "trusted_recurring_users": 8.666799294102633,
-        "log_transaction_count": 4.441688461052498
+        "log_transaction_count": 4.441688461052498,
+        "impact_index": 1.0
```
Comment (swellander) [inline packages/nextjs/pages/api/etl/impact_metric_weights.json:20]: Added `impact_index` here artificially to satisfy our TS compiler

File: packages/nextjs/pages/api/etl/index.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
-import weightingsJSON from "./weightings.json";
+import weightData from "./impact_metric_weights.json";
```
Comment (swellander) [inline packages/nextjs/pages/api/etl/index.ts:1]: Pulling in the actual weights, now that our metric names line up

**Review by escottalexander — CHANGES_REQUESTED** [review]
This is great. Appreciate you doing this. One issue is that we will still need the mapping. The source for the metric names was a static data set that had the mapping already added to each project but in the future we will pull the data from a different source and it is unlikely that this data will include the mapping data. Feel free to reach out for more clarity.

**Review by escottalexander — APPROVED** [review]
LGTM!

**Pull request merged**